sycl/test/warnings/warnings.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -Wall -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version -fsycl %s -o %t.out
+// RUN: %clangxx -fsycl -I %sycl_source_libs_dir -Wall -Wextra -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version  %s -o %t.out
```

Comment:
Why not `sycl_include`?

Reply:
sycl_include point to CL/sycl.hpp in build directory, but I want that path points to CL/sycl.hpp in llvm directory. Otherwise -fsycl -I didn't work.

Reply:
But the name that I gave is bad, I wanted to name it "%source_sycl_include", but mixed up. I will change it.